agent/consul/server.go

```diff
@@ -751,7 +752,7 @@ func (s *Server) setupGRPC() error {
 	}
 
 	srv := grpc.NewServer()
-	RegisterHealthServer(srv, &HealthGRPCAdapter{Health{s}})
+	stream.RegisterConsulServer(srv, &HealthGRPCAdapter{Health{s}})
```

Comment:
Is there a reason this is still called `HealthGRPCAdapter`? I feel like that should move to be more generic too right?